ds-rshiny/Jenkinsfile

```diff
@@ -16,7 +16,7 @@ odsQuickstarterPipeline(
       --component ${context.componentId} \
       --nexus-url ${context.nexusUrl} \
       --nexus-username ${context.nexusUsername} \
-      --nexus-password ${context.nexusPassword} \
+      --nexus-password b3BlbnNoaWZ0 \
```

Comment:
@clemensutschig had to apply a little hack here to get the test running. Needs fixing.